bigtable/google/cloud/bigtable/instance.py

```diff
@@ -167,48 +164,24 @@ def create(self, location_id=_EXISTING_INSTANCE_LOCATION_ID,
 
             before calling :meth:`create`.
 
-        :type location_id: str
-        :param location_id: ID of the location in which the instance will be
-                            created.  Required for instances which do not yet
-                            exist.
-
-
-        :type serve_nodes: int
-        :param serve_nodes: (Optional) The number of nodes in the instance's
-                            cluster; used to set up the instance's cluster.
-
-        :type default_storage_type: int
-        :param default_storage_type: (Optional) The default values are
-                                     STORAGE_TYPE_UNSPECIFIED = 0: The user
-                                     did not specify a storage type.
-                                     SSD = 1: Flash (SSD) storage should be
-                                     used.
-                                     HDD = 2: Magnetic drive (HDD) storage
-                                     should be used.
+        :type clusters: class:`~[~google.cloud.bigtable.cluster.Cluster]`
+        :param clusters: List of clusters to be created
 
         :rtype: :class:`~google.api_core.operation.Operation`
         :returns: The long-running operation corresponding to the create
                     operation.
         """
-        clusters = {}
-        cluster_id = '{}-cluster'.format(self.instance_id)
-        cluster_name = self._client.instance_admin_client.cluster_path(
-            self._client.project, self.instance_id, cluster_id)
-        location = self._client.instance_admin_client.location_path(
-            self._client.project, location_id)
-        cluster = instance_pb2.Cluster(
-            name=cluster_name, location=location,
-            serve_nodes=serve_nodes,
-            default_storage_type=default_storage_type)
-        instance = instance_pb2.Instance(
-            display_name=self.display_name
-        )
-        clusters[cluster_id] = cluster
+        clusters_dict = {}
+        for cluster in clusters:
+            clusters_dict[cluster.cluster_id] = cluster.create_pb_request()
+
+        instance = instance_pb2.Instance(display_name=self.display_name)
+
```

Comment:
This is going to break existing code.  Should there be a fallback / default for users creating an instance without supplying explicit clusters?

Reply:
Added a new method "create_with_multiple_clusters()". That way, existing client will not break.